#intentional error

list = ['Joshua tree', 'Grand Teton']
print(list[2])

'''IndexError: list index out of range
PS C:\Users\leabi\Downloads\CSC 141> & C:/Users/leabi/AppData/Local/Programs/Python/Python312/python.exe "c:/Users/leabi/Downloads/CSC 141/03_introducing_lists/3_11_Intentional_Error.py"
Traceback (most recent call last):
  File "c:\Users\leabi\Downloads\CSC 141\03_introducing_lists\3_11_Intentional_Error.py", line 4, in <module>
    print(list[2])
          ~~~~^^^
IndexError: list index out of range
PS C:\Users\leabi\Downloads\CSC 141> '''